definitions/ext-pihole/tests/Log.json

```diff
@@ -0,0 +1,8 @@
+[
+	{
+		"piHoleName": "myPiHole"
+	},
+	{
+		"metricName": "pihole_count"
+	}
```

Comment:
These data point generates a validation error, thus the PR is rejected when tested locally.